civis/resources/_resources.py

```diff
@@ -404,9 +405,25 @@ def parse_path(path, operations, api_version, resources):
 
 
 def parse_swagger(swagger, api_version, resources):
-    """ Parse a swagger specifiction into a dictionary of classes
+    """ Dynamically create classes to interface with the Civis API.
+
+    Parse an OpenAPI (Swagger) specification into a dictionary of classes
     where each class represents an endpoint resource and contains
     methods to make http requests on that resource.
+
+    Parameters
+    ----------
+    api_spec : OrderedDict
```

Comment:
I'm also fine switching the docstring to say "swagger". If you think it's worth another PR to improve the language, I'm happy to review it.